secret.json

```diff
@@ -0,0 +1,4 @@
+[ { "name": "brandons_wallet",
```

Comment:
This should not be added to git.

Reply:
Sorry, I didn't recognize this was added to git commit.  This was removed.